wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -798,6 +798,47 @@ public final class MLSService: MLSServiceInterface {
 
     typealias PendingJoin = (groupID: MLSGroupID, epoch: UInt64)
 
+    public func establishPendingGroup(groupID: MLSGroupID) async throws {
+        guard let context else {
+            return
+        }
+        let conversation = await context.perform {
+            ZMConversation.fetch(with: groupID, in: context)
+        }
+
+        guard let conversation else {
+            throw MLSServiceError.conversationNotFound
+        }
+
+        try await internalEstablishPendingGroup(
+            groupID: groupID,
+            pendingGroup: conversation,
+            context: context
+        )
+
+        await save(context)
```

Comment:
Move to use case?

Reply:
for now let's leave it as it is, the mlsService is injected in the useCase but not the context which is accessible only via a couple of intermediate objects.